tests/foreman/ui/test_packages.py

```diff
@@ -123,3 +131,68 @@ def test_positive_check_package_details(self):
                     ['Build Time', '1331831364'],
                 ]
             )
+
+
+class RHPackagesTestCase(UITestCase):
+    """Implement tests for packages via UI"""
+
+    @classmethod
+    @skip_if_not_set('fake_manifest')
+    def setUpClass(cls):
+        super(RHPackagesTestCase, cls).setUpClass()
+        cls.organization = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(cls.organization.id, manifest.content)
+        repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=cls.organization.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        entities.Repository(id=repo_id).sync()
+
+    @tier3
```

Comment:
because tier ideology has nothing to do with complexity =)
It is System test, because we create subscription, synchronize real repository and then do content search inside repository for specific package
At least three-four application areas are affected, so I made decision like that